### ISSUE TITLE
 Error:  "uniq" is not exported by the List::Util module

### ISSUE THREAD

**Issue by pscott-au** [pscott-au/WebService-GoogleAPI-Client#6]
**Describe the bug**
Build with older version of List::Util Fails

**To Reproduce**
```
Running make test
PERL_DL_NONLAZY=1 /usr/bin/perl "-MExtUtils::Command::MM" "-e" "test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
t/00-load.t .............. 1/?
#   Failed test 'use WebService::GoogleAPI::Client;'
#   at t/00-load.t line 25.
#     Tried to use 'WebService::GoogleAPI::Client'.
#     Error:  "uniq" is not exported by the List::Util module
# Can't continue after import errors at /home/bizzon/.cpan/build/WebService-GoogleAPI-Client-0.11-ONjjGe/blib/lib/WebService/GoogleAPI/Client/Discovery.pm line 12.
# BEGIN failed--compilation aborted at /home/bizzon/.cpan/build/WebService-GoogleAPI-Client-0.11-ONjjGe/blib/lib/WebService/GoogleAPI/Client/Discovery.pm line 12.
# Compilation failed in require at /home/bizzon/.cpan/build/WebService-GoogleAPI-Client-0.11-ONjjGe/blib/lib/WebService/GoogleAPI/Client.pm line 12.
# BEGIN failed--compilation aborted at /home/bizzon/.cpan/build/WebService-GoogleAPI-Client-0.11-ONjjGe/blib/lib/WebService/GoogleAPI/Client.pm line 12.
# Compilation failed in require at t/00-load.t line 25.
# BEGIN failed--compilation aborted at t/00-load.t line 25.
Bailout called.  Further testing stopped:
# Looks like you failed 1 test of 2.
FAILED--Further testing stopped.
make: *** [test_dynamic] Ошибка 1
  LOCALSHOP/WebService-GoogleAPI-Client-0.11.tar.gz
  /usr/bin/make test -- NOT OK
//hint// to see the cpan-testers results for installing this module, try:
  reports LOCALSHOP/WebService-GoogleAPI-Client-0.11.tar.gz
Running make install
  make test had returned bad status, won't install without force
Failed during this command:
 LOCALSHOP/WebService-GoogleAPI-Client-0.11.tar.gz: make_test NO
```

**Expected behavior**
make to complete without error

**Comment by pscott-au** [comment]
List::Util is a core module but the uniq function was not provided until v1.45 which was included in an older version of Perl around 5.8 ish

**Comment by pscott-au** [comment]
https://github.com/pscott-au/WebService-GoogleAPI-Client/commit/afe41f54aadbd7f7d85c34ff1877e59e3c60cbd2
